examples/webhooks/webhooks_test.go

```diff
@@ -74,14 +80,20 @@ func TestNewEvent(t *testing.T) {
 				EventID:   uuid.NewString(),
 				EventType: tt.eventType,
 				Data:      dataBytes,
-				CreatedOn: time.Now().UTC(),
+				CreatedOn: createdOn,
 			}
 
 			var body bytes.Buffer
 			err = json.NewEncoder(&body).Encode(event)
 			require.NoError(t, err)
 
 			req := httptest.NewRequest(http.MethodPost, "/my-awesome-webhook-url", &body)
+			req.Header.Set("x-timestamp", timestamp)
```

Comment:
Why not break out the header testing in another test so its easier to document and present how it all comes together? I worry this test of parsing events is going to get way to long and become unreadable.